src/seedu/addressbook/AddressBook.java

```diff
@@ -365,37 +381,42 @@ private static void loadDataFromStorage() {
      * @return  feedback about how the command was executed
      */
     private static String executeCommand(String userInputString) {
-        final String[] commandTypeAndParams = splitCommandWordAndArgs(userInputString);
-        final String commandType = commandTypeAndParams[0];
-        final String commandArgs = commandTypeAndParams[1];
-        switch (commandType) {
+        final String[] COMMAND_TYPE_AND_PARAMS = splitCommandWordAndArgs(userInputString);
+        final String COMMAND_TYPE = COMMAND_TYPE_AND_PARAMS[0];
+        final String COMMAND_ARGS = COMMAND_TYPE_AND_PARAMS[1];
+        return specifyCommand(COMMAND_TYPE, COMMAND_ARGS);
+    }
+
+	private static String specifyCommand(final String COMMAND_TYPE,
+			final String COMMAND_ARGS) {
+		switch (COMMAND_TYPE) {
         case COMMAND_ADD_WORD:
-            return executeAddPerson(commandArgs);
+            return executeAddPerson(COMMAND_ARGS);
         case COMMAND_FIND_WORD:
-            return executeFindPersons(commandArgs);
+            return executeFindPersons(COMMAND_ARGS);
         case COMMAND_LIST_WORD:
             return executeListAllPersonsInAddressBook();
         case COMMAND_DELETE_WORD:
-            return executeDeletePerson(commandArgs);
+            return executeDeletePerson(COMMAND_ARGS);
         case COMMAND_CLEAR_WORD:
             return executeClearAddressBook();
         case COMMAND_HELP_WORD:
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
         default:
-            return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
+            return getMessageForInvalidCommandInput(COMMAND_TYPE, getUsageInfoForAllCommands());
         }
-    }
+	}
 
     /**
      * Splits raw user input into command word and command arguments string
      *
      * @return  size 2 array; first element is the command type and second element is the arguments string
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
-        final String[] split =  rawUserInput.trim().split("\\s+", 2);
-        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+        final String[] SPLIT =  rawUserInput.trim().split("\\s+", 2);
```

Comment:
this is not a constant though. constants are usually scalar datatypes, not like arrays.